### PR TITLE
Cut 0.0.110

### PR DESCRIPTION
**Pull request by TheBlueMatt**
Still waiting on at least one more PR, and maybe more, so these will change, but draft release notes for 110.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,50 @@
...
+ * `ProbabilisticScoringParameters::banned_nodes` has been renamed
+   `manual_node_penalties` and changed to take msat penalties (#1592).
+ * Per-payment tracking of failed paths was added to enable configuration of
+   `ProbabilisticScoringParameters::considered_impossible_penalty_msat` (#1600)
+ * `ProbabilisticScoringParameters::base_penalty_amount_multiplier_msat` was
+   added to allow a penalty that is only amount-dependent (#1617).
+ * `ProbabilisticScoringParameters::amount_penalty_multiplier_msat` was renamed
+   `liquidity_penalty_amount_multiplier_msat` (#1617).
+ * Support has been added for DNS hostnames in the `NetAddress` type, as
+   described in BOLT PR 911 (#1553).
```
Comment (tnull) [inline CHANGELOG.md:16]: Maybe link the BOLT PR here? Also, do we generally want to start linking the referenced issues?

```suggestion
   described in [BOLT PR #911](https://github.com/lightning/bolts/pull/911) (#1553).
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,50 @@
...
+ * `GossipSync` now has `rapid`, `p2p`, and `none` constructors (#1618).
+ * `lightning-net-tokio` no longer requires types to be in `Arc`s (#1623).
+ * The `htlc_maximum_msat` field is now required in `ChannelUpdate` gossip
+   messages. In tests this rejects < 1% of channels (#1519).
+ * `ReadOnlyNetworkGraph::{channel,node}` have been added to query for
+   individual channel/node data, primarily for bindings users (#1543).
+ * `FeeEstimator` implementations are now wrapped internally to ensure values
+   below 253 are never used (#1552).
+ * Route selection no longer attempts to randomize path selection, this is
```
Comment (tnull) [inline CHANGELOG.md:25]:
```suggestion
 * Route selection no longer attempts to randomize path selection. This is
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,50 @@
...
+   unlikely to lead to a material change in the paths selected (#1610).
+
+## Bug Fixes
+ * Fixed a panic when deserializing `ChannelDetails` objects (#1588).
+ * When routing, channels are no longer fully saturated before MPP splits are
```
Comment (tnull) [inline CHANGELOG.md:30]: Oh, do we really consider saturating a bug?

**Review by dunxen — APPROVED** [review]
Changelog LGTM
(Not sure if anything else missing)

File: lightning-background-processor/Cargo.toml [hunk 4]
```diff
@@ -15,10 +15,10 @@ rustdoc-args = ["--cfg", "docsrs"]
...
-lightning-rapid-gossip-sync = { version = "0.0.109", path = "../lightning-rapid-gossip-sync" }
+lightning = { version = "0.0.110", path = "../lightning", features = ["std"] }
+lightning-rapid-gossip-sync = { version = "0.0.110", path = "../lightning-rapid-gossip-sync" }
 
 [dev-dependencies]
-lightning = { version = "0.0.109", path = "../lightning", features = ["_test_utils"] }
-lightning-invoice = { version = "0.17.0", path = "../lightning-invoice" }
-lightning-persister = { version = "0.0.109", path = "../lightning-persister" }
+lightning = { version = "0.0.110", path = "../lightning", features = ["_test_utils"] }
+lightning-invoice = { version = "0.18.0", path = "../lightning-invoice" }
```
Comment (valentinewallace) [inline lightning-background-processor/Cargo.toml:23]: Was there a reason to increment the lightning-invoice version? Or do we always increment it?

**Review by valentinewallace — COMMENTED** [review]
Looks good after squash!

**Review by valentinewallace — APPROVED** [review]

**Pull request merged**